### PR TITLE
Revive part 2 of the netherium monster corpse project

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Content "Finish part 2 of the netherium monster corpse"

#### Purpose of change
I noticed that #54155 had been closed and marked as a desired PR, so I decided to finish it. 

#### Describe the solution
Finish the planned content from #54155 in the "Purpose of Change" category, and have it spawn. See the PR for full details.  

#### Describe alternatives you've considered
Not doing this or leaving it to someone else to do.

#### Testing
I spawned everything multiple times in testing to make sure it all worked, and everything worked great.

#### Additional context
I changed the C++ for the admire effect into a JSON-based EOC. Additionally, I did not include the finishing textures for the organs in the final draft, as I am unfamiliar with the processing of tilesets.

### CODE REVIEW

**Comment by Raikiri** [comment]
This is so cool, I wish we had more stuff to do in the endgame like this.

**Comment by andrei8l** [comment]
The build job chokes on the space in `data/json/mapgen/nether_monster_corpse/monster nested.json`'name. Use an underscore instead of space.

**Comment by gkarfakis19** [comment]
Hey. I was the original author of the netherum corpse project but real life circumstances have made me unable to work on this any longer. It is very exciting to see other people trying to finish what I started (in retrospect, it was a bit *too* ambitious). Please let me know if you need any clarifications on what I did, I'd be happy to help.

**Comment by MNG-cataclysm** [comment]
@gkarfakis19 What was your original plan for the project overall?

**Comment by gkarfakis19** [comment]
@MNG-cataclysm 
My original idea was to have these netherum monsters undeground in the overworld that were pretty hard to find, and included a variety of interesting nether creatures inside. Loot would mainly include body parts from the creatures or the monster (perhaps something mutagenic in nature, and/or interesting chemical ingredients).
Original plan was 3 PR's. One was the overmap generation, which I am quite satisfied with, the second one (which you're building on) would include the internal terrain shape of the creature + the furniture/walls inside it, with a few internals as proof of concept, and the third one would finally populate it with the internal creatures and items.
I think the concept is actually pretty damn hard to do, and you would need a close collaboration between whoever is designing the internals and whoever is doing the actual art for them for it to look natural. I tried to do some graphics work (you will find the linked Ultica PR on the original PR) to get a very basic idea of what it would look like, but I am a terrible artist and couldn't really make a lot of progress.
I think the PR is still a cool idea, but it's probably really hard to finish as I originally planned. Feel free to reduce the scope of this if you feel it is necessary.

**Comment by MNG-cataclysm** [comment]
I think it could be done currently, especially using the work you've done with it. You did a great job. 

Did you make any sort of design document for it that could be referenced?

**Comment by gkarfakis19** [comment]
Unfortunately I didn't make a design doc for this, but honestly I haven't given this project much more thinking than what I've typed in here, so I don't think it would have been too helpful even if I had.

File: data/json/furniture_and_terrain/furniture_nether_monster.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    "coverage": 50,
+    "required_str": -1,
+    "flags": [ "TRANSPARENT" ],
+    "emissions": [ "emit_monster_gas_1", "emit_monster_gas_2" ]
+  },
+  {
+    "type": "furniture",
+    "id": "f_nm_nether_lily",
+    "name": "beautiful nether lily",
+    "//~": "It's thing is making itself irrestible to the PC. TODO: Make this effect more obvious",
```
Comment (TheShadowFerret) [inline data/json/furniture_and_terrain/furniture_nether_monster.json:33]:
```suggestion
    "//~": "Its thing is making itself irresistible to the PC. TODO: Make this effect more obvious",
```

**Comment by Fris0uman** [comment]
Last useless commit was to try and add gkarfakis19 as co-author but I must have messed the spacing or something in the commit message becasue it didn't work, my bad ^^"

**Pull request merged**